### ISSUE TITLE
Add Issue Template and PR Template

### ISSUE THREAD

**Issue by Devanshukoli** [Dipeshdahiya/Remoteflow#12]
Hey! I would love to add the *issue templates* and *PR Templates* for this project. Such that anyone can create issue and PR without much hitch.

**Comment by Dipeshdahiya** [comment]
can you please specify the work you wanna do

**Comment by Devanshukoli** [comment]
Yes, You see, currently when creating any **Issues** in this project, there is not a template for , such that anyone can easily create any issues. 
GitHub as it's own Issue Templates and PR Templates. I would like to add into this project.

**Comment by Dipeshdahiya** [comment]
issue assigned
